### DISCUSSION TITLE
Setting Cache Driver

### DISCUSSION THREAD

**Discussion by Gex2501** [alexjustesen/speedtest-tracker#1555] [Q&A]
In the [documentation](https://docs.speedtest-tracker.dev/other/caching) it mentions using database or redis for caching. But how do I specify my redis connection? I've found some mention in code about REDIS_HOST and REDIS_PASSWORD but has anyone actually set this up? I'd really like a way to confirm it's working. 

Also, where are the logs kept? I've used the 'docker logs' command but there is very little info there after startup.

**Comment by alexjustesen** [comment]
You can use any of the Laravel supported cache drivers, while database is the default you could also use Redis or Memcached: https://laravel.com/docs/11.x/cache#configuration

Set the following vars so hook into your Redis instance:

```
# cache driver
CACHE_DRIVER: redis

# redis config
REDIS_HOST:http(s)://path-to-host
REDIS_PASSWORD:null # unless you have a password
REDIS_PORT:6379
```
